packages/react-slider/src/components/Slider/Slider.test.tsx

```diff
@@ -122,11 +122,10 @@ describe('Slider', () => {
     expect(screen.getByRole('slider').getAttribute('type')).toEqual('range');
   });
 
-  it('applies ariaValueText', () => {
+  it('applies aria-valuetext', () => {
```

Comment:
we decided to remove the imperative aria text helper and simply pass value into aria-valueText